boards/xtensa/esp32s2/esp32s2-saola-1/scripts/.gitignore

```diff
@@ -1 +0,0 @@
-/esp32s2_out.ld
```

Comment:
why not removing `.gitignore` at all?

Reply:
It was removed:

![image](https://user-images.githubusercontent.com/32426024/199844040-d70f4753-85e9-4a99-8b12-09115d825e0a.png)